packages/client/components/NewMeetingActionsCurrentMeetings.tsx

```diff
@@ -77,8 +75,8 @@ export default createFragmentContainer(NewMeetingActionsCurrentMeetings, {
         meetingType
         id
         createdAt
-        facilitator {
-          id
+        createdBy
+        createdByUser {
```

Comment:
optionally, try using `readInlineData` in `useSnacksForNewMeetings` & then you can replace this stuff with a fragment spread. that way, the data is being used in the same file that it is requested in

(`readInlineData` is a new-ish function that lets you request things in functions outside of a component. it's kinda nice in cases like this